frontend/src/pages/__snapshots__/ResourceSelector.test.tsx.snap

```diff
@@ -29,7 +29,7 @@ exports[`ResourceSelector displays resource selector 1`] = `
     emptyMessage="Test - Sorry, no resources."
     filterLabel="test filter label"
     initialSortColumn="created_at"
-    isCalledByV1={true}
+    isCalledByV1={false}
```

Comment:
While this does work for listing v2 pipelines, I'm guessing it breaks it for v1 pipelines. I'm trying to wrap my head around why this `isCalledByV1` exists, because you can either listPipelines based off v1 api or v2 api, but not both, yet can have v1/v2 pipelines uploaded together in KFP, which makes me think that we actually don't properly support listing/filtering for both kfp v1 and v2 pipelines together 

@zijianjoy does that sound right to you?

either way I would rather we have this work with v2 rather than v1, and discuss a better design approach on how to handle this case for when we have both pipelines uploaded and need to list/filter simultaneously

@zijianjoy wdyt?

Reply:
isCalledByV1 was introduced to differentiate API request payload schema between V1 API and V2 API. https://github.com/kubeflow/pipelines/blob/1c7954de261b22fa93905cf11b6b280fdf19c320/frontend/src/components/CustomTable.tsx#L229-L246. Because we have a flag to control whether to show v2 UI by default during the initial migration.

Ideally v2 should support filer for V1 names filtering as well. So it makes sense to me for switching to V2 filter mechanism.